src/Compilers/CSharp/Portable/BoundTree/BoundNodes.xml

```diff
@@ -2074,6 +2072,21 @@
   <Node Name="BoundDiscardPattern" Base="BoundPattern">
   </Node>
 
+  <!-- PROTOTYPE(list-patterns): UNDONE: Base node for BoundListPattern as well as BoundDeclarationPattern and BoundRecursivePattern -->
```

Comment:
Do we really need anything more specific than `BoundPattern`?

Reply:
It's just to reuse Variable and VariableAccess fields (currently duplicated for all three)